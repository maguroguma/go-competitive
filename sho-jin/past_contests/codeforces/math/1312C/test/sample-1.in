5
4 100
0 0 0 0
1 2
1
3 4
1 4 1
3 2
0 1 3
3 9
0 59049 810
