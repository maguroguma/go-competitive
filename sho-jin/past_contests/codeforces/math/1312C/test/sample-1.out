YES
YES
NO
NO
YES
